experiments/benchmarks/bucket.jl

```diff
@@ -24,6 +24,7 @@ import ClimaUtilities.TimeVaryingInputs: TimeVaryingInput
 import ClimaTimeSteppers as CTS
 import NCDatasets
 using ClimaCore
+ClimaCore.Operators.use_fd_shmem() = false
```

Comment:
```suggestion
import ClimaCore
ClimaCore.Operators.use_fd_shmem() = false
```
Ah, this may actually be important. We need to `import` ClimaCore, as `using` gives no indication to julia that we want to override behavior.

Reply:
Ill fix this!